src/install.ts

```diff
@@ -111,7 +113,15 @@ class UI {
       common.installLatest(this);
   }
 
-  get clangdPath(): string { return config.get<string>('path'); }
+  get clangdPath(): string {
+    const p = config.get<string>('path');
```

Comment:
nit: const => let, a const can not be changed.

Reply:
Done, and actually tested this time...